modules/aleph/src/Aleph/AlephClient.php

```diff
@@ -251,6 +251,45 @@ public function getItems(AlephMaterial $material) {
     );
   }
 
+  /**
+   * Create payment.
+   *
+   * @param \Drupal\aleph\Aleph\Entity\AlephPatron $patron
+   *   The Aleph patron.
+   * @param int $sum
+   *   Amount paid.
+   * @param string $reference
+   *   Identification of the payment.
+   *
+   * @return \SimpleXMLElement|false
+   *   The SimpleXMLElement from the raw XML response.
+   *
+   * @throws AlephClientException
+   */
+  public function addPayment(AlephPatron $patron, $sum, $reference) {
+    $options = array(
+      'query' => array('institution' => 'ICE53'),
```

Comment:
This hardcodes `ICE53` instead of using a property. Is that intentional? Code comments would suggest that the `$filter_institutions` argument to the constructor would contain this value. However it is not assigned to a object property at the moment 🤔 .

Reply:
`ICE53` is used elsewhere, the question is whether it's logically the same thing. We're using it for "which library top pay debts at" which is bound to where they actually are registered. Why the request method is using it is the question. Obviously for filtering, but the "why" is not clear.

Reply:
See https://github.com/rvk-utd/ding2/pull/187/commits/151ee9a148495eb526fc9cbb66b2c37911882d78